lib/doro/behaviors/turntable.ex

```diff
@@ -29,27 +29,8 @@ defmodule Doro.Behaviors.Turntable do
 
   defp set_playing(object, playing) do
     Doro.World.set_prop(object, :playing, playing)
-    update_description(object, playing)
   end
 
   defp is_playing?(turntable), do: turntable.props[:playing]
 
-  defp update_description(turntable, true) do
```

Comment:
i'll come back to this - how to update description based on the entities props...